erpc_c/transports/erpc_rpmsg_lite_base_transport.h

```diff
@@ -54,6 +66,12 @@ class RPMsgBaseTransport : public Transport
 protected:
     static struct rpmsg_lite_instance *s_rpmsg; /*!< Pointer to instance of RPMSG lite. */
     static uint8_t s_initialized;               /*!< Represent information if the rpmsg-lite was initialized. */
+#if RL_USE_STATIC_API
+    struct rpmsg_lite_instance m_static_context;
+    struct rpmsg_lite_ept_static_context m_ept_context;
+    rpmsg_static_queue m_queue_context;
```

Comment:
rpmsg_static_queue -> rpmsg_static_queue_ctxt as per latest changes in the rpmsg_lite repo